core/src/subgraph/registrar.rs

```diff
@@ -411,7 +411,7 @@ where
         hash: &DeploymentHash,
         node_id: &NodeId,
     ) -> Result<(), SubgraphRegistrarError> {
-        let locations = self.store.locators(hash)?;
+        let locations = self.store.locators(hash, true)?;
```

Comment:
We expect this to return a vector with a single `DeploymentLocator` (the active copy), right?